README.md

```diff
@@ -1,33 +1,83 @@
+and it is highly recommended that you set it up when developing a new feature.
 
-There is a `chart/` directory, which is a one-to-one copy of the default deployment chart supplied by GitLab, **except** that the service account we created above is automatically bound to the deployment, so it can manage jobs.
+In order to connect with Inframock, follow the instructions in here next: https://github.com/biomage-ltd/inframock
 
-This will need to be automated and simplified in due course.
+After Inframock is started, the next step is to start the API.
 
-Local development
-=================
+### 2. Start the API
+Whether the API runs with the local InfraMock instance, which should have Redis as well as all the mocked AWS 
+services ready for use, or with a ***live*** cluster instead, is controlled by an enviornment variable called
+`CLUSTER_ENV`, which is set to `development` by default.
+
+Run `npm start` to run the API locally, while running Inframock from the previous step in a separate terminal tab.
+The output on the terminal should look similarly to this:
 
-You can use InfraMock to develop locally. To do so, make sure you have InfraMock set up and running
-in the background. You should be able to simply start the API and it should default to the local InfraMock
-instance, which should have Redis as well as all the mocked AWS services ready for use.
+```
+[2020-12-18T07:06:20.852Z] We are running on a development cluster, patching AWS to use InfraMock endpoint...
+[2020-12-18T07:06:21.097Z] Generating configuration for cache...
+[2020-12-18T07:06:21.097Z] Attempting to fetch URLs for Redis cluster endpoints...
+[2020-12-18T07:06:21.097Z] Running locally, keeping base configuration.
+[2020-12-18T07:06:21.098Z] Primary: localhost:6379, reader: localhost:6379
+[2020-12-18T07:06:21.098Z] Setting up L1 (in-memory) cache, size: 1000, TTL: 129600000
+[2020-12-18T07:06:21.098Z] Now setting up Redis connections...
+[2020-12-18T07:06:21.098Z] Running in development, patching out TLS connection.
+[2020-12-18T07:06:21.100Z] Running in development, patching out TLS connection.
+[2020-12-18T07:06:21.100Z] Cache instance created.
+[2020-12-18T07:06:21.104Z] NODE_ENV: development, cluster env: development
+[2020-12-18T07:06:21.104Z] Server listening on port: 3000
+[2020-12-18T07:06:21.108Z] redis:primary Connection successfully established.
+[2020-12-18T07:06:21.109Z] redis:reader Connection successfully established.
+[2020-12-18T07:06:21.111Z] redis:primary Connection ready.
+[2020-12-18T07:06:21.112Z] redis:reader Connection ready.
+```
 
 To use a **live** (`staging` or `production`) cluster, make sure you run the API with the `CLUSTER_ENV` environment
-variable set to the appropriate value.
+variable set to the appropriate value. For example, running `CLUSTER_ENV='production' npm start` will connect you to the
+production cluster and as a result you should see an output on the terminal similar to this:
```

Comment:
this actually didn't work for me. After running `CLUSTER_ENV='production' npm start` I also got:

```
[2020-12-18T07:20:16.662Z] redis:reader An error occurred: connect ETIMEDOUT
```

a couple of times. Is it not possible to connect to the redis cluster in production anymore? @marcellp can you try to do it and if it is not possible, I will just remove those instructions.

Reply:
It is possible to connect to the production cluster, just not to Redis directly, i.e. you will lose the caching ability.

Reply:
cool, I will make a note of that